cmd/kubeadm/app/phases/controlplane/manifests.go

```diff
@@ -40,7 +40,11 @@ import (
 const (
 	DefaultCloudConfigPath = "/etc/kubernetes/cloud-config"
 
-	defaultv17AdmissionControl = "Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,DefaultTolerationSeconds,NodeRestriction,ResourceQuota"
+	// admission controllers for any version <= current stable release
+	// for kubeadm this is the current stable release and and 1 major release behind
+	defaultStableAdmissionControl = "Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,DefaultTolerationSeconds,NodeRestriction,ResourceQuota"
+	// admission controllers > current stable release
+	defaultUnstableAdmissionControl = "Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,DefaultTolerationSeconds,NodeRestriction,ResourceQuota,Priority"
```

Comment:
I'm not sure where in the chain this should be, but I'm pretty sure it should be early in the chain. Please search for how it's set up in GCE/GKE

Reply:
Good catch, will use this as a referene https://github.com/kubernetes/kubernetes/pull/49322/files#diff-57380bb03c3598646d5a3cb66518fa0bL237